### PR TITLE
[Halloween] Jacq fixes 2020 edition

### PR DESCRIPTION
**Pull request by Thalpy**
## About The Pull Request

Maintains Jacqueline the Pumpqueen in time for her event, so that you don't have any bugs/runtimes while she's about.
Since this is her first year here, I set her dialogue to work of the first year's stuff. It should auto update for next year when it comes by!

## Why It's Good For The Game

Maintains code that's about to be active very soon!
Add runechat for jacq's unique way of talking
Fixes some of her walking bugs
Fixes her poof() to stop runtime spamming if it doesn't get lucky
Ensure any ghostroles who go on dates as her don't get kicked out
Updates her dialogue for this year and next

Basically, maintains Jacq.

## Changelog
:cl:
tweak: tweaked Jacq's dialogue
fix: fixes Jacq's tricks, treats and poofs
/:cl:

### CODE REVIEW

**Comment by SarmentiCampbell** [comment]
Oh hell yeah, Dating Simulator is now fixed

**Review by Azarak — APPROVED** [review]
